### PR TITLE
HIVE-26988: Apache website add redirects to search engine cached pages

### PR DESCRIPTION
**Pull request by simhadri-g**
…ch engine cached page.

### CODE REVIEW

**Review by ayushtkn — COMMENTED** [review]
Makes sense to me.
One more thing around the URL stuff:
https://hive.apache.org//community/mailinglists/

It shows double slash before community and couple of more pages when I click on them, is that something that can be solved?

cc. @zabetak

**Comment by simhadri-g** [comment]
Yes.

**Comment by simhadri-g** [comment]
@zabetak , sorry I think i removed you as reviewer for the PR by mistake

File: content/Developement/gettingStarted.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: "GettingStarted"
 date: 2023-01-10T12:35:11+05:30
 draft: false
+aliases: [/GettingStarted]
```
Comment (zabetak) [inline content/Developement/gettingStarted.md:5]: Was there ever a link:
`https://hive.apache.org/GettingStarted`?

I just tried the following and does not work either.
`https://apache.github.io/hive-site/GettingStarted`

Comment (zabetak) [reply content/Developement/gettingStarted.md:5]: I think we don't need an alias. We just need to fix:
https://github.com/apache/hive-site/blob/6097c3f2cf8e0186a39f172ac8165509b1457a6e/content/Developement/gettingStarted.md?plain=1#L55

to point to the wiki:
https://cwiki.apache.org/confluence/display/Hive/GettingStarted

Comment (simhadri-g) [reply content/Developement/gettingStarted.md:5]: You are right, aliases is not needed for the Getting started page. We need these for the other pages at least until google claws the hive website and updates its search index.

Pointing the wiki to https://cwiki.apache.org/confluence/display/Hive/GettingStarted  is being addressed here by Mahesh - https://issues.apache.org/jira/browse/HIVE-26983

Thanks!

**Pull request closed without merging**